workitem/workitem_repository_blackbox_test.go

```diff
@@ -35,290 +31,222 @@ func TestRunWorkItemRepoBlackBoxTest(t *testing.T) {
+	})
 }
 
-func (s *workItemRepoBlackBoxTest) TestExistsWorkItem() {
-	t := s.T()
-	resource.Require(t, resource.Database)
-
-	t.Run("work item exists", func(t *testing.T) {
+func (s *workItemRepoBlackBoxTest) TestCreate() {
+	s.T().Run("ok - save assignees", func(t *testing.T) {
 		// given
-		wi, err := s.repo.Create(
-			s.Ctx, s.space.ID, workitem.SystemBug,
-			map[string]interface{}{
-				workitem.SystemTitle:       "Title",
-				workitem.SystemDescription: rendering.NewMarkupContentFromLegacy("Description"),
-				workitem.SystemState:       workitem.SystemStateNew,
-			}, s.creator.ID)
-		require.Nil(s.T(), err, "Could not create workitem")
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(1, func(fxt *tf.TestFixture, idx int) error {
+			fxt.WorkItems[idx].Fields[workitem.SystemAssignees] = []string{"A", "B"}
+			return nil
+		}))
 		// when
-		err = s.repo.CheckExists(s.Ctx, wi.ID.String())
+		wi, err := s.repo.LoadByID(s.Ctx, fxt.WorkItems[0].ID)
 		// then
 		require.Nil(t, err)
+		require.Len(t, wi.Fields[workitem.SystemAssignees].([]interface{}), 2)
+		assert.Equal(t, "A", wi.Fields[workitem.SystemAssignees].([]interface{})[0])
+		assert.Equal(t, "B", wi.Fields[workitem.SystemAssignees].([]interface{})[1])
 	})
 
-	t.Run("work item doesn't exist", func(t *testing.T) {
-		t.Parallel()
+	s.T().Run("ok - create work item with description no markup", func(t *testing.T) {
+		// given
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(1, func(fxt *tf.TestFixture, idx int) error {
+			fxt.WorkItems[idx].Fields[workitem.SystemDescription] = rendering.NewMarkupContentFromLegacy("Description")
+			return nil
+		}))
 		// when
-		err := s.repo.CheckExists(s.Ctx, "00000000-0000-0000-0000-000000000000")
+		wi, err := s.repo.LoadByID(s.Ctx, fxt.WorkItems[0].ID)
 		// then
-
-		require.IsType(t, errors.NotFoundError{}, err)
+		require.Nil(t, err)
+		// workitem.WorkItem does not contain the markup associated with the description (yet)
+		assert.Equal(t, rendering.NewMarkupContentFromLegacy("Description"), wi.Fields[workitem.SystemDescription])
 	})
 
-}
-
-func (s *workItemRepoBlackBoxTest) TestCreateWorkItemWithDescriptionMarkup() {
-	// given
-	wi, err := s.repo.Create(
-		s.Ctx,
-		s.space.ID,
-		workitem.SystemBug,
-		map[string]interface{}{
-			workitem.SystemTitle:       "Title",
-			workitem.SystemDescription: rendering.NewMarkupContent("Description", rendering.SystemMarkupMarkdown),
-			workitem.SystemState:       workitem.SystemStateNew,
-		},
-		s.creator.ID)
-	require.Nil(s.T(), err, "Could not create workitem")
-	// when
-	wi, err = s.repo.LoadByID(s.Ctx, wi.ID)
-	// then
-	require.Nil(s.T(), err)
-	// workitem.WorkItem does not contain the markup associated with the description (yet)
-	assert.Equal(s.T(), rendering.NewMarkupContent("Description", rendering.SystemMarkupMarkdown), wi.Fields[workitem.SystemDescription])
-}
+	s.T().Run("ok - work item with description markup", func(t *testing.T) {
```

Comment:
Should there be one more test for `SystemTitle with markup`? They are different attributes but has the same behavior.